lib/mock_redis/set_methods.rb

```diff
@@ -27,6 +27,11 @@ def sadd(key, members)
       end
     end
 
+    def sadd?(key, members)
+      sadd_result = sadd(key, members)
+      sadd_result.is_a?(Numeric) ? sadd_result > 0 : sadd_result
```

Comment:
shouldn't this always return a `boolean` ? - I'd love to get this in as well ! :-)